diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 02c7a2ff9..81f785fff 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -12944,7 +12944,7 @@ static void task_change_group_fair(struct task_struct *p)
 	 * We couldn't detach or attach a forked task which
 	 * hasn't been woken up by wake_up_new_task().
 	 */
-	if (READ_ONCE(p->__state) == TASK_NEW)
+	if (READ_ONCE(p->state) == TASK_NEW)
 		return;
 
 	detach_task_cfs_rq(p);
